### PR TITLE
feat: Added 2 inputs for permit papers

### PR DESCRIPTION
**Pull request by Merkur39**

### CODE REVIEW

File: src/components/CompositeHeader/CompositeHeader.jsx [hunk 1]
```diff
@@ -55,7 +57,14 @@ const CompositeHeader = ({
         ))}
       {Text &&
         (isValidElement(Text) || isArray(Text) ? (
-          <div className={'u-mv-1'}>{Text}</div>
+          <div
+            className={cx({
+              ['u-mv1']: !isMobile,
```
Comment (Ldoppea) [inline src/components/CompositeHeader/CompositeHeader.jsx:62]: shouldn't this be `u-mv-1` ?

File: src/locales/en.json [hunk 2]
```diff
@@ -141,11 +141,11 @@
       "obtentionDate": {
         "text": "Fill in the date of each category obtained"
       },
-      "CobtentionDate": {
-        "text": "Truck - depuis le"
+      "CObtentionDate": {
+        "inputLabel": "Truck - since the"
```
Comment (Ldoppea) [inline src/locales/en.json:145]: Not sure about the context but are you sure we should have `the` here?

Comment (Ldoppea) [reply src/locales/en.json:145]: Also why did we remove `depuis le` in french but not in english?

**Review by Ldoppea — APPROVED** [review]

File: src/locales/en.json (hunk 2 above)
Comment (Merkur39) [reply src/locales/en.json:145]: Indeed, we will just keep the name for the English version (for the moment)

**Pull request merged**